src/utils/dateUtils.ts

```diff
@@ -74,6 +74,11 @@ export function isValidDate(date: Date): boolean {
     return false;
   }
 }
-export function dateIsBefore(first: Date, second: Date): boolean {
+
+export function isAfter(first: Date, second: Date): boolean {
```

Comment:
Why rename this? I liked how it was called `dateIsBefore` because it made it clear(er) that it only looks at the date, not the time.

Reply:
oops I think this was a mistake that I commited this, I sometimes rename functions/variables while working because the names confuse me personally, and change them back before committing heh